### ISSUE TITLE
ActiveAdmin 1.3.0 is incompatible with Rails 5.2.1

### ISSUE THREAD

**Issue by portwood** [activeadmin/activeadmin#5447]
When attempting to update a Rails application using the 'bundle update' command, the bundler does not update to Rails 5.2.1 due to ActiveAdmin dependencies.

Inside the Gemfile: 
  gem 'rails', '~> 5.2.0'
  ...
  gem 'activeadmin'

### Expected behavior
The bundle command would resolve the dependencies and install Rails 5.2.1.

### Actual behavior
The bundle command does not install Rails 5.2.1.

What actually happens?
No update.
Running gem dependency 'activeadmin' returns:
Gem activeadmin-1.3.0
  arbre (>= 1.1.1)
  coffee-rails (>= 0)
  formtastic (~> 3.1)
  formtastic_i18n (>= 0)
  inherited_resources (>= 1.7.0)   
  jquery-rails (>= 4.2.0)
  kaminari (>= 0.15)
  railties (< 5.3, >= 4.2)
  ransack (~> 1.3)
  sass (~> 3.1)
  sprockets (< 4.1)

inherited_resources seems to be the issue.

**Comment by rylanb** [comment]
Same issue for me

**Comment by richpeck** [comment]
When you push it to Heroku, you get the "meta_search" error as well.

**Comment by skylerwshaw** [comment]
This has been updated in `inherited_resources` gem. https://github.com/activeadmin/inherited_resources/pull/449 was merged 10 days ago and https://github.com/activeadmin/inherited_resources/pull/451 is on its way through travis-ci at this moment.

**Comment by varyonic** [comment]
@portwood can you confirm this is resolved with the release of inherited_resources 1.9.0?

**Comment by rylanb** [comment]
Worked for me locally!

**Comment by richpeck** [comment]
Working locally, testing on Heroku

**Comment by richpeck** [comment]
Works on Heroku too

**Comment by richpeck** [comment]
![untitled](https://user-images.githubusercontent.com/1104431/43977854-3c01686e-9cdd-11e8-9b6e-314099dcae64.png)

Looks like there's a problem - any insights?

**Comment by jarydkrish** [comment]
@richpeck you should bump the version of `ransack` you are using (basically just make sure you have `ransack 2.0.0`):

```
bundle update ransack
```

Ransack just released support for Rails 5.2.1 in [this PR](https://github.com/activerecord-hackery/ransack/pull/938)

**Comment by portwood** [comment]
I am now experiencing the same issue as @richpeck.

**Comment by jarydkrish** [comment]
@portwood try changing your gemfile to match against the latest master:
```rb
# Gemfile
git_source(:github) do |repo_name|
  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?('/')
  "https://github.com/#{repo_name}.git"
end

gem 'activeadmin', github: 'activeadmin', ref: 'c301ab126b3fac1962edeac124eb46e885217420'
```

Right now [`activeadmin 1.3.0` is pinned to `ransack ~> 1.3`](https://rubygems.org/gems/activeadmin/versions/1.3.0)

So I'm not sure, but there might need to be a new release with a different gemspec.

**Comment by portwood** [comment]
That seems to work. I've made a quick pass through the application and have run my rspec suite without issue. Will test it further.

**Comment by varyonic** [comment]
Just to be clear, I'm not expecting further fixes to Ransack 2.x to require additional changes to AA 1.3.1. If no further feedback I will close in a while.

**Comment by gregmolnar** [comment]
@varyonic The only issue reported for ransack 2.0.0 has been fixed in 2.0.1

**Comment by varyonic** [comment]
Thanks
